cocktails9/cocktails9/Views/login.swift

```diff
@@ -0,0 +1,130 @@
+import SwiftUI
+
+struct login: View {
+    
+    @State private var currentImage = "loginbw2"
+    @State private var email = ""
+    @State private var password = ""
+    @State private var showAlert = false
+    @State private var alertMessage = ""
+    @State private var navigateToGrid = false
+    
+    var body: some View {
+        NavigationView {
+            VStack(spacing: 15) {
+                Spacer(minLength: 0)
+                
+                HStack {
+                    Text("Lo")
+                        .font(.largeTitle)
+                        .fontWeight(.bold)
+                        .padding(.trailing, -7.0)
+                    Text("Gin")
+                        .font(.largeTitle)
+                        .fontWeight(.bold)
+                        .foregroundColor(Color(hue: 0.277, saturation: 0.318, brightness: 0.558))
```

Comment:
good thing here would be to define/add a `Color Set` in `Assets` file and use it here instead of defining a color here with magic numbers.
```suggestion
                        .foregroundColor(Color("green"))
```

I can see it's reused for Login / Register button.